pydov/search/abstract.py

```diff
@@ -623,7 +623,6 @@ def _search(self, location=None, query=None, return_fields=None,
                 filter_request = etree.tostring(filter_request,
                                                 encoding='unicode')
             except LookupError:
-                # Python2.7 without lxml uses 'utf-8' instead.
```

Comment:
I think the whole try/except block can be removed here. This holds true for all similar changes below as well.